Gems/Atom/Feature/Common/Code/Source/AuxGeom/FixedShapeProcessor.cpp

```diff
@@ -354,30 +373,29 @@ namespace AZ
             using NormalType = AuxGeomNormal;
 
             // 1st pole vertex
-            positions.push_back(PosType(0.0f, 0.0f, radius));
-            normals.push_back(NormalType(0.0f, 0.0f, 1.0f));
+            positions.push_back(PosType(0.0f, radius, 0.0f));
```

Comment:
This changes the mesh from z-up to y-up. I believe z-up is the current o3de approach.

Reply:
I needed this change for the rotation with the input direction to work. I can change the default direction passed by the sphere so that the sphere remains Z-up.